textproc/lilypond/Portfile

```diff
@@ -17,113 +16,96 @@ long_description    Lilypond is a unix-based automated engraving system that \
                     similar to LaTeX. Lilypond can export sheet music to PDF, \
                     EPS, SVG, and PNG formats, and can also create MIDI files.
 
-homepage            http://lilypond.org/
-platforms           darwin
-master_sites        http://download.linuxaudio.org/lilypond/sources/v${branch}/
-#dist_subdir         ${name}/${version}_${revision}
-checksums           rmd160  a355201334fd0beac36c96f844c75d8e27b4ea9d \
-                    sha256  329d733765b0ba7be1878ae3f457dbbb875cc2840d2b75af4afc48c9454fba07
+homepage            https://lilypond.org
+master_sites        https://lilypond.org/download/sources/v2.20/
```

Comment:
Ack. I chose to use `set branch 2.20` and `version ${branch}.0` instead of the previous construction of `version 2.20.0` and `set branch [join [lrange [split ${version} .] 0 1] .]` because I feel it is more readable; if you feel strongly in favor of continuing to use the previous construction I can do that instead.